apps/api/Dockerfile

```
@@ -0,0 +1,9 @@
+FROM python:3.9
+
+COPY . .
+
+RUN apt-get update && apt-get upgrade -y
+RUN apt-get install -y pkg-config libxml2-dev libxmlsec1-dev libxmlsec1-openssl
```

Issue: the separation here could cause issues with Docker.
> Using `apt-get` update alone in a `RUN` statement causes caching issues and subsequent `apt-get install` instructions to fail. Always combine RUN apt-get update with apt-get install in the same RUN statement. ([Best Practices for Dockerfile Instructions: apt-get](https://docs.docker.com/develop/develop-images/instructions/#apt-get))

Also don't think we need to run `apt-get upgrade` or install `pkg-config`?

```suggestion
RUN apt-get update && apt-get install -y libxml2-dev libxmlsec1-dev libxmlsec1-openssl
```

---

Ah, I was looking at the pip `xmlsec` package, which had `pkg-config` as one of the necessary libraries. I'll remove it.